com.unity.render-pipelines.universal/Runtime/DeferredLights.cs

```diff
@@ -291,6 +296,15 @@ public DeferredLights(Material tileDepthInfoMaterial, Material tileDeferredMater
             m_Tilers = new DeferredTiler[DeferredConfig.kTilerDepth];
             m_TileDataCapacities = new int[DeferredConfig.kTilerDepth];
 
+            GBufferDescriptors = new AttachmentDescriptor[GBufferSliceCount + 1]; // + Depth
+            GBufferDescriptors[0] = new AttachmentDescriptor(GetGBufferFormat(GBufferAlbedoIndex));
+            GBufferDescriptors[1] = new AttachmentDescriptor(GetGBufferFormat(GBufferSpecularMetallicIndex));
+            GBufferDescriptors[2] = new AttachmentDescriptor(GetGBufferFormat(GBufferNormalSmoothnessIndex));
+            GBufferDescriptors[4] = new AttachmentDescriptor(RenderTextureFormat.Depth);
+#if (UNITY_IOS || UNITY_ANDROID) && !UNITY_EDITOR
```

Comment:
This needs to match GBufferSliceCount. Maybe just use `if (GBufferSliceCount == 5)`